src/core_plugins/timelion/server/handlers/__tests__/parse_sheet.js

```diff
@@ -0,0 +1,23 @@
+const parseSheet = require('../lib/parse_sheet');
+
+const expect = require('chai').expect;
+
+describe('timelion parse_sheet function', function () {
```

Comment:
we do test for the white space behavior in the test below (it should not be allowed)

or do you mean testing that \n and \r are also accepted as white space ?